nextcloudappstore/core/views.py

```diff
@@ -54,6 +58,13 @@ def get_queryset(self):
             queryset = queryset.filter(categories__id=category_id)
         if featured == "true":
```

Comment:
Because if we simply check if `featured` is truthy, `?featured=false` becomes "show only featured apps", because the string `"false"` is truthy. `maintainer` on the other hand may be any username, so we just check if one was provided.